fenix/app/src/test/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsFragmentReducerTest.kt

```diff
@@ -102,10 +105,11 @@ class QuickSettingsFragmentReducerTest {
                 tab = null,
                 url = "https://www.firefox.com",
                 isTrackingProtectionEnabled = true,
-                isCookieBannerHandlingEnabled = true,
+                cookieBannerUIMode = CookieBannerUIMode.ENABLE,
                 listTrackers = listOf(),
                 mode = Normal,
                 lastAccessedCategory = "",
+                siteDomainToReport = "https://www.amazon.de",
```

Comment:
Could you help me to clarify, I'm a bit confused about why do we need `siteDomainToReport` if we already have `url`. Do we have an use case where the user is on site and we are submitting a request for different one?

Reply:
Done